<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab { max-width: 180px; }</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>556</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewTab"/>
    <addaction name="actionNewWindow"/>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionRecentFile0"/>
    <addaction name="actionRecentFile1"/>
    <addaction name="actionRecentFile2"/>
    <addaction name="actionRecentFile3"/>
    <addaction name="actionRecentFile4"/>
    <addaction name="actionCloseTab"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionAddBookmark"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuRotateView">
     <property name="title">
      <string>Rotate &amp;View</string>
     </property>
     <addaction name="actionRotateLeft"/>
     <addaction name="actionRotateRight"/>
    </widget>
    <widget class="QMenu" name="menuPageNavigation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Page &amp;Navigation</string>
     </property>
     <addaction name="actionFirstPage"/>
    </widget>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>&amp;Zoom</string>
     </property>
     <addaction name="actionZoomTo"/>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
    </widget>
    <addaction name="menuRotateView"/>
    <addaction name="menuPageNavigation"/>
    <addaction name="separator"/>
    <addaction name="menuZoom"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutGather"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>&amp;User</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionAccount"/>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuUser"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenFile">
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Co&amp;py</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>P&amp;aste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate &amp;Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate &amp;Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionAboutGather">
   <property name="text">
    <string>About &amp;Gather</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRecentFile0">
   <property name="text">
    <string>Recent File &amp;1</string>
   </property>
  </action>
  <action name="actionRecentFile1">
   <property name="text">
    <string>Recent File &amp;2</string>
   </property>
  </action>
  <action name="actionRecentFile2">
   <property name="text">
    <string>Recent File &amp;3</string>
   </property>
  </action>
  <action name="actionRecentFile3">
   <property name="text">
    <string>Recent File &amp;4</string>
   </property>
  </action>
  <action name="actionRecentFile4">
   <property name="text">
    <string>Recent File &amp;5</string>
   </property>
  </action>
  <action name="actionNewTab">
   <property name="text">
    <string>New &amp;Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionNewWindow">
   <property name="text">
    <string>&amp;New Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>&amp;Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionFirstPage">
   <property name="text">
    <string>&amp;First Page</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoomTo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom &amp;To</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Boo&amp;kmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </action>
  <action name="actionAccount">
   <property name="text">
    <string>&amp;Account</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
